src/DataTable/RowsPerPageInput/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
```

Comment:
I use folder name as component name, and name the component file index.js , then I can put other related files into this folder, such as css files, StoryBook files, testing files, and logic files. I didn't apply this pattern through out the project, I just want to make a demo here :)